CredScanSuppressions.json

```diff
@@ -16,6 +16,7 @@
         "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_azconfig_feature_filter.yaml",
         "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_azconfig_import_export_naming_conventions.yaml",
         "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_azconfig_import_export.yaml",
+        "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_azconfig_key_validation.yaml",
```

Comment:
@haroldrandom please look at the credential scan issue.

Reply:
![image](https://user-images.githubusercontent.com/14357159/72773648-c4791880-3c42-11ea-808f-160aeba048bd.png)
Test PR: https://github.com/Azure/azure-cli/pull/11911/checks?check_run_id=400111935
Build Raw Log: https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/239104/logs/193

Credential Suppression Confirmed.